Artifacts/windows-powershellcore/installPowerShellCore.ps1

```diff
@@ -0,0 +1,79 @@
+﻿param
+(
+    [Parameter(Mandatory = $true)]
+    [String]
+    $packageUrl,
+
+    [Parameter()]
+    [bool]
+    $installCRuntime = $false
+)
+
+if (-not (Split-Path -Path $packageUrl -Leaf).EndsWith('.msi'))
+{
+    throw "${packageUrl} is not the path to PowerShell Core MSI."
+}
+
+try
+{
+    $coreMSI = "${env:Temp}\PowerShellCore.msi"    
+    Write-Verbose -Message "Downloading ${packageUrl} to ${coreMSI} ..."
```

Comment:
Consider just using `Write-Host`. No real need to hide such a message under a verbose flag.